zone/quest_parser_collection.cpp

```diff
@@ -959,13 +965,16 @@ QuestInterface* QuestParserCollection::GetQIByNPCQuest(uint32 npc_id, std::strin
 	);
 
 	std::vector<std::string> file_names = {
-		fmt::format("{}/{}", zone_versioned_path, npc_id), // Local versioned by NPC ID ./quests/zone/v0/10.ext
-		fmt::format("{}/{}", zone_versioned_path, npc_name), // Local versioned by NPC Name ./quests/zone/v0/npc.ext
+		fmt::format("{}/{}", zone_versioned_path, npc_id), // Local versioned by NPC ID
+		fmt::format("{}/{}", zone_versioned_path, npc_name), // Local versioned by NPC Name
```

Comment:
Why'd we remove the zone_versioned_path comments ? They are helpful to new travelers

Reply:
Added the comments back in